uvotskycorr.py

```diff
@@ -38,11 +37,11 @@
     outfile = filename.replace("sk.img", "aspcorr.ALL")
     attfile = filename.split('_',1)[0] + "pat.fits"
     terminal_output_file = path + "output_uvotskycorrID_" + filename.replace('.img','.txt')
-    catfile = "/home/mdcleir/Documents/SWIFT_datareduction_pipeline/V2/usnob1.spec"
+    catfile = os.getcwd() + "/usnob1.spec"
 
     #Open the terminal output file and run uvotskycorr ID with the specified parameters:
     with open(terminal_output_file,"w") as terminal:
-        subprocess.call("uvotskycorr what=ID skyfile=" + skyfile + " corrfile=NONE attfile=" + attfile + " outfile=" + outfile + " starid='matchtol=20 cntcorr=3 n.reference=200 n.observation=40 max.rate=1000' catspec=" + catfile + " chatter=5", cwd=path, shell=True, stdout=terminal)
+        subprocess.call("uvotskycorr what=ID skyfile=" + skyfile + " corrfile=NONE attfile=" + attfile + " outfile=" + outfile + " starid='matchtol=20 cntcorr=3 n.reference=200 n.observation=40 max.rate=1000' catspec=" + catfile + " chatter=5 clobber=yes", cwd=path, shell=True, stdout=terminal)
```

Comment:
i think `clobber=yes` is okay here, but it's not clear to me that we need it. i'm wondering if i should remove it.

Reply:
According to my notes, the `clobber` keyword is not actually doing anything in this function. Even when you put `clobber=no`, the correction is still applied to the original file. However, here only the calculation of the correction is done, and not the application. This step writes the aspect corrections to a separate file. In short, I don't think the `clobber` keyword is needed here, so I would remove it, especially because it was not there originally. Once we have the tests in place, we can do a quick check whether or not this keyword makes a difference.

Reply:
what if the aspect correction was already created?

when i added this clobber=yes i think it was because i was trying to run this step of the pipeline a second time and it was failing without `clobber=yes`

we can revisit this later though.

Reply:
i removed the clobber=yes for now

Reply:
This is something we should investigate when you are making all steps idempotent. See issue #8.